src/types/comfy.d.ts

```diff
@@ -101,7 +101,7 @@ export interface ComfyExtension {
 export type ComfyObjectInfo = {
   name: string
   display_name?: string
-  description?: string
+  description?: [string | string, string | string, string, Record<string, any>]
```

Comment:
`export type DescriptionSpec = z.infer<zDescriptionSpec>` in apiTypes.ts and use it here.

Reply:
Much cleaner. Thanks.